addons/pos_restaurant/static/tests/tours/FloorScreen.tour.js

```diff
@@ -36,6 +36,20 @@ registry.category("web_tour.tours").add("FloorScreenTour", {
             FloorScreen.clickTable("1"),
             FloorScreen.selectedTableIs("1"),
 
+            //test copy floor
+            FloorScreen.clickFloor("Main Floor"),
+            FloorScreen.clickEditButton("Copy"),
+            {
+                content: `Verify floor is copy`,
+                trigger: 'div.floor-selector button:contains("Main Floor (copy)")',
```

Comment:
```suggestion
                trigger: 'div.floor-selector button.btn-primary:contains("Main Floor (copy)")',
```
this is in order to check that we are on the correct floor